app/controllers/admin/deploy_groups_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::DeployGroupsController < ApplicationController
+  before_action :authorize_admin!
+  before_action :authorize_super_admin!, only: [ :create, :new, :update, :destroy ]
+  before_action :deploy_group, only: [:edit, :update, :destroy]
```

Comment:
Aye, there is magic with rails afoot with not needing to declare this method explicitly.